tests/unittests/Foundation/NSOperationTests.mm

```diff
@@ -88,14 +88,15 @@ - (instancetype)initWithBlock:(void (^)())block;
 @implementation BlockThread
 - (instancetype)initWithBlock:(void (^)())block {
     if (self = [super init]) {
-        self->_block = block;
-        self->_finishCondition = [[_NSBooleanCondition new] autorelease];
+        self.block = block;
```

Comment:
er, not sure what you're getting excited about here, actually. if a block were observable it'd be on the block member changing, not the block finishing.